docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md

```diff
@@ -643,7 +643,12 @@ Of course the same is possible by adding a boolean variable to a configuration o
 
 Builtin variables can be leveraged to apply a patch only for a specific Kubernetes version.
 ```yaml
-    enabledIf: "{{ if eq "v1.21.1" .builtin.controlPlane.version }}true{{end}}"
+    enabledIf: '{{ semverCompare "1.21.1" .builtin.controlPlane.version }}'
```

Comment:
Very nice. thank you!

Just to confirm, `semverCompare` directly returns true so the if => true is not necessary, correct?

Reply:
That is correct.

The sprig Docs do not properly describe the return value for `semverCompare`, but sprig [calls](https://github.com/Masterminds/sprig/blob/05f1d18231b62a3cc05e4babc33f0f6dbf933f7f/semver.go#L18) [`github.com/Masterminds/semver.Constraints#Check`](https://github.com/Masterminds/semver/blob/49c09bfed6adcffa16482ddc5e5588cffff9883a/constraints.go#L56) which returns `bool`.

Reply:
I opened an upstream PR to improve the docs around this: Masterminds/sprig#325

Reply:
Perfect! Thank you